app/components/find/courses/qualifications_summary_component/view.html.erb

```diff
@@ -9,16 +9,13 @@
         It may also allow you to teach overseas, though you should always check what qualifications are needed in the country you’d like to teach in.
       </p>
     <% end %>
-<% when "PGCE with QTS" %>
-    <%= govuk_details(summary_text: "PGCE with QTS") do %>
+<% when "QTS with PGCE" %>
+    <%= govuk_details(summary_text: "QTS with PGCE") do %>
       <p class="govuk-body">
-        A postgraduate certificate in education (PGCE) with qualified teacher status (QTS) will allow you to teach in state schools in England and may allow you to teach in other parts of the UK.
+       You need qualified teacher status (QTS) to teach in state schools in England. QTS may also allow you to teach in other parts of the UK.
       </p>
       <p class="govuk-body">
-        It may also allow you to teach overseas, though you should always check what qualifications are needed in the country you’d like to teach in.
-      </p>
-      <p class="govuk-body">
-        Many PGCE courses include credits that count toward a Master’s degree.
+        This course also offers a postgraduate certificate in education (PGCE). PGCE courses can include credits that count towards a master’s degree.
       </p>
     <% end %>
   <% when "PGDE with QTS" %>
```

Comment:
Should we be updating "PGDE with QTS" to "QTS with PGDE" as well?

Reply:
> Should we be updating "PGDE with QTS" to "QTS with PGDE" as well?

I asked the team and they mentioned PGDE will be done in a separate card and separate work.

Reply:
Fair enough - seems like an unnecessary split to me 🤷‍♂️